week-2/stretch/till.js

```diff
@@ -8,11 +8,11 @@ function totalTill(till) {
   let total = 0;
 
   for (const [coin, quantity] of Object.entries(till)) {
-    console.log(coin * quantity);
-    total += coin * quantity;
+   // console.log(coin * quantity);
```

Comment:
The function is nicely written.
Could meet something similar in our production code base at work.

Reply:
Thanks